keyboards/narfkb/rules.mk

```diff
@@ -0,0 +1,2 @@
+SERIAL_DRIVER = vendor
+ENCODER_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
would the `#ifdef ENCODER_ENABLE` in narfkb.c still function if this change was implemented?

Reply:
yes